src/style.css

```diff
@@ -0,0 +1,53 @@
+* {
+  margin: 0;
+}
+
+.star {
+  width: 100px;
+  height: 96px;
+}
```

Comment:
Are you sure you need total container size hardcoded since you provide size of individual element later?